frontend/src/components/custom/QuestionTable/QuestionTable.tsx

```diff
@@ -8,14 +8,24 @@ function QuestionTable() {
   const [data, setData] = useState<Question[]>([]);
   const [loading, setLoading] = useState(true); // To show a loading state
 
+  // Define a refetch function to reload the questions
+  const refetch = async () => {
+    setLoading(true);
+    const questions = await getData();
+
+    questions.sort((a, b) => {
+      return (
+        new Date(a.dateCreated).getTime() - new Date(b.dateCreated).getTime()
+      );
+    });
```

Comment:
Do we need to sort here if we have already sorted it in `getData()`?

Reply:
true okay will update